src/timeout.ts

```diff
@@ -76,10 +100,16 @@ export class Timeout extends Promise<never> {
   clear(): void {
     clearTimeout(this.id);
     this.id = undefined;
+    this.timedOut = false;
+    this.cleared = true;
+  }
+
+  throwIfExpired(): void {
+    if (this.timedOut) throw new TimeoutError('Timed out', { duration: this.duration });
   }
 
-  public static expires(durationMS: number, unref?: boolean): Timeout {
-    return new Timeout(undefined, durationMS, unref);
+  public static expires(duration: number, unref?: true): Timeout {
+    return new Timeout(undefined, { duration, unref });
   }
 
   static is(timeout: unknown): timeout is Timeout {
```

Comment:
We do not use this is, remove it along with the toStringTag, less code to maintain